docs/kh2/file/type/preferences.md

```diff
@@ -0,0 +1,284 @@
+
+| File | Description |
+|--------|---------------|
+| plyr 	 | Player
+| fmab 	 | Form Abilities
+| prty 	 | Party
+| sstm 	 | System
+| magi 	 | Magic
+
+### plyr
+Each pointer leads to a specific entry's offset.
+
+### plyr Structure
+
+| Amount | Description |
+|--------|---------------|
+| 1 	 | Pointer count [uint]
+| 59 	 | Pointers [uint]
+| 10 	 | Plyr entries
+
+### plyr Entry
+
+| Offset | Variable Type | Description |
+|--------|---------------|-------------|
+| 0 | float | Attack Y Offset
+| 4 | float | Attack Radius
+| 8 | float | Attack Min H
+| 12 | float | Attack Max H
+| 16 | float | Attack V Angle
+| 20 | float | Attack V Range
+| 24 | float | Attack S Range
+| 28 | float | Attack H Angle
+| 32 | float | Attack U Min H
+| 36 | float | Attack U Max H
+| 40 | float | Attack U Range
+| 44 | float | Attack J Front
+| 48 | float | Attack Air Min H
+| 52 | float | Attack Air Max H
+| 56 | float | Attack Air Big H Offset
+| 60 | float | Attack UV0
+| 64 | float | Attack JV0
+| 68 | float | Attack First V0
+| 72 | float | Attack Combo V0
+| 76 | float | Attack Finish V0
+| 80 | float | Blow Recov H
+| 84 | float | Blow Recov V
+| 88 | float | Blow Recov Time
+| 92 | float | Auto Lock On Range
+| 96 | float | Auto Lock On Min H
+| 100 | float | Auto Lock On Max H
+| 104 | float | Auto Lock On Time
+| 108 | float | Auto Lock On H Adjust
+| 112 | float | Auto Lock On Inner Adjust
+
+
+### fmab
+Each pointer leads to a specific entry's offset.
+
+### fmab Structure
+
+| Amount | Description |
+|--------|---------------|
+| 1 	 | Pointer count [uint]
+| 5 	 | Pointers [uint]
+| 5 	 | fmab entries
+
+### fmab Entry
+
+| Offset | Variable Type | Description |
+|--------|---------------|-------------|
+| 0 | float | High Jump Height
+| 4 | float | Air Dodge Height
+| 8 | float | Air Dodge Speed
+| 12 | float | Air Slide Time
+| 16 | float | Air Slide Speed
+| 20 | float | Air Slide Brake
+| 24 | float | Air Slide Stop Brake
+| 28 | float | Air Slide Star Time
+| 32 | float | Glide Speed
+| 36 | float | Glide Fall Ratio
+| 40 | float | Glide Fall Height
+| 44 | float | Glide Fall Max
+| 48 | float | Glide Accel
+| 52 | float | Glide Start Height
+| 56 | float | Glide End Height
+| 60 | float | Glide Turn Speed
+| 64 | float | Dodge Roll Star Time
+
+### prty
+Each pointer leads to a specific entry's offset.
+
+### prty Structure
+
+| Amount | Description |
+|--------|---------------|
+| 1 	 | Pointer count [uint]
+| 70 	 | Pointers [uint]
+| 5 	 | prty entries
+
+### prty Entry
+
+| Offset | Variable Type | Description |
+|--------|---------------|-------------|
+| 0 | float | Walk Speed
+| 4 | float | Run Speed
+| 8 | float | Jump Height
+| 12 | float | Turn Speed
+| 16 | float | Hang Height
+| 20 | float | Hang Margin
+| 24 | float | Stun Time
+| 28 | float | Mp Drive
+| 32 | float | Up Down Speed
+| 36 | float | Dash Speed
+| 40 | float | Accel
+| 44 | float | Brake
+| 48 | float | Subjective Offset
+
+### sstm
```

Comment:
Can you add a linefeed after all of the titles? GitHub is known for breaking the formatting with titles without linefeeds.

Reply:
Woops, fixed.